lib/Event/PostDocumentRenderEvent.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\RST\Event;
+
+use Doctrine\Common\EventArgs;
+use Doctrine\RST\Configuration;
+use Doctrine\RST\Nodes\DocumentNode;
+use Doctrine\RST\Templates\TemplateRenderer;
+
+use function sprintf;
+
+/**
+ * This event is called in the DocumentNodeRenderer after the content of
+ * the document got rendered and before the HTML is returned. It can be used
+ * to postprocess or exchange the HTML that will be written to the file.
```

Comment:
Did you really mean to write "exchange"? To me, you exchange something with something else.

```suggestion
 * to postprocess or change the HTML that will be written to the file.
```